.github/workflows/ci.yml

```diff
@@ -23,76 +24,99 @@ jobs:
       - name: Check out code
         uses: actions/checkout@master
       - name: Run yaml Lint
-        uses: actionshub/yamllint@main
+        uses: actionshub/yamllint@master
 
-  # Vagrant tests are very slow and often time out, so only check those suites that
-  # don't run as root across latest centos, debian and ubuntu.
-  vagrant:
-    runs-on: macos-latest
+  mdl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@master
+      - name: Run Markdown Lint
+        uses: actionshub/markdownlint@master
+
+  dokken:
+    needs: [mdl, yamllint, delivery]
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         os:
-          - 'centos-8'
-          - 'debian-10'
-          - 'ubuntu-2004'
+          - amazonlinux-2
+          - debian-9
+          - debian-10
+          - centos-7
+          - centos-8
+          - ubuntu-1804
+          - ubuntu-2004
         suite:
-          - 'client-runas-splunk'
-          - 'server-runas-splunk'
-          # Run the server in Vagrant too, as it splunk often does not keep running in Dokken.
+          - 'client'
+          - 'uninstall-forwarder'
+          - 'client-inputs-outputs'
           - 'server-runas-root'
+          - 'server-cluster-master'
+          - 'server-shdeployer'
+          - 'server-shcluster-member'
+          - 'disabled'
+          - 'upgrade-server'
+          - 'upgrade-client'
+          - 'server-resources'
+          - 'client-resources'
       fail-fast: false
 
     steps:
       - name: Check out code
         uses: actions/checkout@master
       - name: Install Chef
-        uses: actionshub/chef-install@main
-      - name: Test Kitchen
-        uses: actionshub/test-kitchen@master
+        uses: actionshub/chef-install@master
+      - name: Dokken
+        uses: actionshub/kitchen-dokken@master
         env:
           CHEF_LICENSE: accept-no-persist
-          KITCHEN_LOCAL_YAML: test/kitchen/kitchen.vagrant.yml
+          KITCHEN_LOCAL_YAML: kitchen.dokken.yml
         with:
           suite: ${{ matrix.suite }}
           os: ${{ matrix.os }}
+      - name: Print debug output on failure
+        if: failure()
+        run: |
+            set -x
+            sudo journalctl -l --since today
+            KITCHEN_LOCAL_YAML=kitchen.dokken.yml /usr/bin/kitchen exec ${{ matrix.suite }}-${{ matrix.os }} -c "journalctl -l"
 
-  # Run most tests in dokken as it's quicker than Vagrant
-  dokken:
-    runs-on: ubuntu-latest
+  vagrant:
+    needs: [mdl, yamllint, delivery]
+    runs-on: macos-latest
     strategy:
       matrix:
         os:
-          - 'centos-7'
-          - 'centos-8'
-          - 'debian-10'
-          - 'debian-9'
-          - 'ubuntu-1604'
-          - 'ubuntu-1804'
-          - 'ubuntu-2004'
+          - amazonlinux-2
+          - debian-9
+          - debian-10
+          - centos-7
+          - centos-8
+          - ubuntu-1804
+          - ubuntu-2004
```

Comment:
This should only list the MacOS OS since we test the others via dokken.

Reply:
This actually needs to test all of these because the runas-splunk recipes aren't working on dokken